pkg/data/components/KeepErrors.yaml

```diff
@@ -0,0 +1,63 @@
+kind: KeepErrors
+name: Keep Errors
+style: sampler
+type: base
+status: development
+version: v0.1.0
+summary: Keeps traces with errors at a specified rate.
+description: |
+  Any trace that contains any span with an error field will be sampled
+  at the specified rate (default 1). The name of the error field can be
+  overridden (default 'error').
+tags:
+  - category:refinery_rule
+  - service:refinery
+  - vendor:Honeycomb
+  - type:error
+ports:
+  - name: In
+    direction: input
+    type: HoneycombEvents
+  - name: Out
+    direction: output
+    type: HoneycombEvents
+properties:
+  - name: Environment
+    summary: The environment in which to enable the sampler.
+    description: |
+      The environment in which to enable the sampler. If this field is
+      not specified, the sampler will be enabled in the __default__ environment,
+      which is what is used if the environment specified in the trace is not found.
+    type: string
+    default: "__default__"
+  - name: FieldName
```

Comment:
Do you think it would be useful to have an array of field names in case there are multiple fields an error could be found? This might be useful for distributed traces where different processors / SDKs use different span attribute keys.